index.js

```diff
@@ -6,22 +6,49 @@ const chalk = require('chalk');
 const prettyBytes = require('pretty-bytes');
 const StreamCounter = require('stream-counter');
 const gzipSize = require('gzip-size');
+const brotliSize = require('brotli-size');
 
 module.exports = options => {
```

Comment:
```suggestion
module.exports = (options = {}) => {
```

Then you could simplify the below check.